compiler/damlc/tests/daml-test-files/ConstrainedClassMethod.daml

```diff
@@ -1,5 +1,3 @@
-{-# LANGUAGE ConstrainedClassMethods #-}
```

Comment:
This is enabled by default.

Reply:
Actually it isn’t. The GHC docs lied to me by claiming that `MultiParamTypeclasses` imply `ConstrainedClassMethods`.